arch/arm/soc/nordic_nrf5/nrf52/Makefile

```diff
@@ -10,6 +10,7 @@ soc-cflags += -DNRF52840_XXAA
 endif
 
 obj-y += soc.o
+obj-$(CONFIG_NRF52_ARM_MPU_ENABLE) += arm_mpu_regions.o
```

Comment:
mpu_regions is the configuration file for a specific mpu, to easily identify the vendor, the format of the configuration file name currently is <vendor>_mpu_regions and the driver filename is <vendor>_mpu.